addons/hr/__manifest__.py

```diff
@@ -54,6 +54,8 @@
         'web.qunit_suite_tests': [
             'hr/static/tests/**/*',
             ('remove', 'hr/static/tests/tours/**/*'),
+            ('remove', 'hr/static/tests/disable_patch.js'),
+            'hr/static/tests/disable_patch.js',
```

Comment:
> I am not sure why we do this ?

This is a suggestion of @xavierbol to avoid having the patch of the user avatar card applied before it is extended to create the employee avatar card. This, combined with the patchWithCleanup used before each test, allows to keep the patch in the employee avatar card in the testing.